hack/util.sh

```diff
@@ -23,6 +23,7 @@ CLUSTERLINK_TARGET_SOURCE=(
   clusterlink-controller-manager=cmd/clusterlink/controller-manager
   clustertree-cluster-manager=cmd/clustertree/cluster-manager
   virtual-cluster-operator=cmd/kubenest/operator
+  node-agent=cmd/kubenest/node-agent/go
```

Comment:
```suggestion
  node-agent=cmd/kubenest/node-agent
```

Reply:
node-agent has golang and python version, so the source directory is `cmd/kubenest/node-agent/go`, change to `cmd/kubenest/node-agent` will lead make error

Reply:
fixed